CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,73 @@
+- Support shared LoadBalancerIP for multiple Services by introducing a new annotation `annotation service.antrea.io/allow-shared-load-balancer-ip: true`. ([#6480](https://github.com/antrea-io/antrea/pull/6480), [@tnqn])
+- Promote feature `CleanupStaleUDPSvcConntrack` from Alpha to Beta. ([#6372](https://github.com/antrea-io/antrea/pull/6372), [@hongliangl])
+- Always include Pod labels in FlowAggregator IPFIX template. ([#6418](https://github.com/antrea-io/antrea/pull/6418), [@antoninbas])
+- Fix live config updates on IPFIXExporter for FlowAggregator. ([#6385](https://github.com/antrea-io/antrea/pull/6385), [@antoninbas])
+- Improve handling of config changes in FlowAggregator to support updating `recordContents.podLabels` at runtime. ([#6378](https://github.com/antrea-io/antrea/pull/6378), [@antoninbas])
+- Add an `EndpointResolver` to remove Antrea Agent's dependency on proxy to access Antrea Service. ([#6361](https://github.com/antrea-io/antrea/pull/6361), [@antoninbas])
+- Replace `bincover` with built-in Go coverage profiling tool. ([#6090](https://github.com/antrea-io/antrea/pull/6090), [@shikharish])
+- Trim unneeded fields stored in informers and Node objects to reduce memory footprint. ([#6317](https://github.com/antrea-io/antrea/pull/6317) [#6351](https://github.com/antrea-io/antrea/pull/6351), [@tnqn])
+- Remove stale multicast routes to improve the readability of multicast routes. ([#3242](https://github.com/antrea-io/antrea/pull/3242), [@ceclinux])
+- Add `srcPodIP` field in Traceflow observations. ([#6247](https://github.com/antrea-io/antrea/pull/6247), [@Atish-iaf])
+- Use Helm to generate Antrea Windows manifests. ([#6360](https://github.com/antrea-io/antrea/pull/6360), [@shikharish])
+- Upgrade CNI plugins from v1.4.1 to v1.5.1. ([#6475](https://github.com/antrea-io/antrea/pull/6475), [@antoninbas])
+- Add documentation for the `sameLabels` feature in Antrea ClusterNetworkPolicy. ([#6280](https://github.com/antrea-io/antrea/pull/6280), [@Dyanngg])
+- Add recommend usage of FQDN policies. ([#6389](https://github.com/antrea-io/antrea/pull/6389), [@Dyanngg])
+
+### Fixed
+
+- Fix NodePortLocal rules being deleted incorrectly due to PodIP recycle. ([#6531](https://github.com/antrea-io/antrea/pull/6531), [@tnqn])
+- Fix "Access is denied" error when importing certificates into the trusted publishers store on Windows 2022. ([#6529](https://github.com/antrea-io/antrea/pull/6529), [@wenyingd])
+- Attach OVS uplink after removing `flow-restore-wait` flag to fix Node network connection breaking issue when `SecondaryNetwork` is enabled. ([#6504](https://github.com/antrea-io/antrea/pull/6504), [@wenyingd])
+- Exclude terminated Pods from group members when calculating `AppliedToGroups` and `EgressGroups` to prevent NetworkPolicy from having no effect when IP is reused. ([#6508](https://github.com/antrea-io/antrea/pull/6508), [@tnqn])
```

Comment:
"Exclude terminated Pods from `AppliedToGroups` and `EgressGroups` calculation to prevent NetworkPolicies or Egresses applying to wrong IPs, when terminated Pod IPs are reused."?

Reply:
@tnqn could you comment? I think the phenomenon is that NetworkPolicies take no effects?

Reply:
Jianjun is correct, but perhaps "prevent NetworkPolicies or Egresses applying to wrong **Pods**" is more accurate.

Reply:
Updated.